src/PromisifiedFS.js

```diff
@@ -69,7 +69,7 @@ module.exports = class PromisifiedFS {
     // but there might not be any other fs operations needed until later. Therefore we
     // need to attempt to release the mutex
     this._activate().then(() => {
-      if (this._operations.size === 0) {
+      if (this._operations.size === 0 && !this._deactivationTimeout) {
```

Comment:
I hope that the identifier is always a non-zero integer :trollface: 
I'd need to look at the spec, but if it's not a hard requirement... :laughing:

Reply:
I looked at the spec. :D

Reply:
> let handle be a user-agent-defined integer that is greater than zero

source: https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html